### PR TITLE
Topic 0 html and css basics

### PR DESCRIPTION
**Pull request by agustinavighetti**

### CODE REVIEW

File: 00-HTML-CSS-basics/i.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/i.html]: Fix indentation here.

File: 00-HTML-CSS-basics/iii.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/iii.html]: Fix indentation here.

Should be:

```
<html>
    <head>
    </head>
    <body>
    [...]
```

File: 00-HTML-CSS-basics/index_2.1.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport" />
+    <meta charset="utf-8">
+    <link href="style_2.1.css" rel="stylesheet" type="text/css">
+    <title>CSS</title>
+</head>
+<body>
+
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/index_2.1.html]: There's an extra blank line here.

File: 00-HTML-CSS-basics/index_2.1.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+<div class="wrapper">
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/index_2.1.html]: Please fix indentation in this file (recurring for HTML files)

File: 00-HTML-CSS-basics/index_2.1.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    <header class="hea1">
+        header1
+    </header>
+    <div class="subWrapper">
+        <nav>
+            nav
+        </nav>
+        <section>
+            <header class="hea2">
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/index_2.1.html]: This class is not following naming conventions regarding BEM. If your div (that it's the block) is called `subWrapper` then this header should be called `subWrapper--header`.

File: 00-HTML-CSS-basics/index_2.3.html [hunk 6]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport" />
+    <meta charset="utf-8">
+    <link href="style_2.3.css" rel="stylesheet" type="text/css">
+
+ <!-- .......ITEM 6........
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/index_2.3.html]: Good enough!

File: 00-HTML-CSS-basics/index_2.3.html [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+    </style> -->
+
+    <title>CSS</title>
+</head>
+<body>
+
+<div class="wrapper">
+    <header class="header">
+        <font>header1</font>
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/index_2.3.html]: Is this `font` needed? I don't think so.

Comment (nawetimebomb) [reply 00-HTML-CSS-basics/index_2.3.html]: It's looks like you used `font` to override `font-size` value for item 6 and then you figured out that you can use asterisk for that... Am I right?

File: 00-HTML-CSS-basics/index_2.5.1.html [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+    </style> -->
+
+    <title>CSS</title>
+</head>
+<body>
+
+<div class="wrapper">
+    <header class="header">
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/index_2.5.1.html]: Same as before with BEM conventions.

File: 00-HTML-CSS-basics/iv.html [hunk 9]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="name=value,name=value" />
+    <meta charset="utf-8">
+    <title>Table</title>
+    <style type="text/css">
+      
+    th {
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/iv.html]: Please, do not use HTML tags. I know this is before CSS but anyway exercise didn't asked for it, is that true? So if you add something extra of what the exercise asks for you should do it the best way you can and you know how to create external stylesheets.

File: 00-HTML-CSS-basics/layout.html [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport">
+    <meta charset="utf-8">
+    <link href="layout.css" rel="stylesheet" type="text/css">
+    <title>Layout</title>
+</head>
+<body>
+
+<!-- PROBLEM: when the mouse pointer in hover the text, the image stays in black and white -->
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/layout.html]: Code-wise this file is pretty well done. I will take a deeper look into it ASAP.

File: 00-HTML-CSS-basics/layout2.6.html [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport">
+    <meta charset="utf-8">
+    <link href="layout2.6.css" rel="stylesheet" type="text/css">
+    <title>Layout</title>
+</head>
+<body>
+
+<!-- BUG: when the mouse pointer is over the text, the image stays in black and white -->
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/layout2.6.html]: Am I seeing duplicate files??

File: 00-HTML-CSS-basics/layout2.6_js.html [hunk 12]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport">
+    <meta charset="utf-8">
+    <link href="layout2.6_js.css" rel="stylesheet" type="text/css">
+<!-- THIS IS JUST TO FIX THE BUG I HAD WITH THE COLOR OF THE IMAGES WHEN THE MOUSE POINTER WAS OVER THE TEXT -->    
+    <script type="text/javascript">
+        function color1() {
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/layout2.6_js.html]: Oh I gotcha! You just created different solutions. Ok!

Please, remember we add a space between method name and parenthesis. `function color1 () {`

File: 00-HTML-CSS-basics/layout2.6_js.html [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+                Follow us on <span class="link"><a href="#">Twitter</a></span>, <span class="link"><a href="#">Facebook</a></span> and
+                <span class="link"><a href="#">Pinterest</a></span> or suscribe to our <span class="link"><a href="#">RSS Feed</a></span>
+            </p>
+        </aside>
+    </div>
+    <div class="images">
+        <div class="col">
+            <div class="img top">
+                <a href="#"><img id="i1" onmouseover="color1()" onmouseout="nocolor1()" src="img1.jpg"></a>
+                <span class="tooltip" onmouseover="color1()" onmouseout="nocolor1()"><a class="tit" href="#">Title</a><br><a class="des" href="#">description</a></span>
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/layout2.6_js.html]: You're using a break line here but in code you're not creating a new line. That's hard to follow.

File: 00-HTML-CSS-basics/v.html [hunk 14]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport" />
+    <meta charset="utf-8">
+    <title>Multimedia</title>
+    <style type="text/css">
+
+        body {
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/v.html]: Same comment as other in-file style tags.

File: 00-HTML-CSS-basics/v.html [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+            display: block;
+            margin: auto;
+            margin-bottom: 100px;
+        }
+        
+    </style>
+</head>
+<body>
+
+<div>
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/v.html]: This need a revision about indentation!

File: 00-HTML-CSS-basics/vi.html [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+                <option>Golf</option>
+                <option>Volleyball</option>
+            </select><br><br>
+        
+        <label>Small bio: </label><br>
+        <textarea cols="50" rows="4"></textarea><br><br>
+        
+        <table>
+            <tr>
+            <th><input class="but" name="submit" type="submit" value="Submit" /></th>
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/vi.html]: Super clean! Good! Some minor things like indentation here.

**Review by nawetimebomb — CHANGES_REQUESTED** [review]
This is a very good work! You got the job done and not a lot of comments in this code review. You'll need to review BEM standards and please be sure that if you add something extra make your effort worth doing it the best you can! Waiting for code updates.

Also, you can just use one normalize.css file!

File: 00-HTML-CSS-basics/layout2.6.html (hunk 11 above)
Comment (agustinavighetti) [reply 00-HTML-CSS-basics/layout2.6.html]: I based on the picture to do the layout.htm, then I saw the actual page with which we had to guide on to do the layout and I saw it had links in the words in bold of the header, in the pictures and in the Title and description of the pics. So, I added all those details in the layout2.6_html. But yes, they are pretty similar.

File: 00-HTML-CSS-basics/index_2.3.html (hunk 7 above)
Comment (agustinavighetti) [reply 00-HTML-CSS-basics/index_2.3.html]: I don't understand what you mean. I did use font for the item 6 to work, otherwise it wouldn't change the font-size, but I don't understand what you mean with the asterisk thing. Should I delete the fonts? Or replace them for other tags?

File: 00-HTML-CSS-basics/layout.css [hunk 17]
```diff
@@ -518,22 +103,6 @@ p {
     z-index:10;
 }
 
-.img:hover span.tooltip::first-line {
+.images--col--img:hover .images--col--img--tooltip::first-line {
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/layout.css]: This is not that cool. We should not have element of an element.

File: 00-HTML-CSS-basics/vi.html [hunk 18]
```diff
@@ -1,92 +1,50 @@
...
+                        <option>Golf</option>
+                        <option>Volleyball</option>
+                    </select><br><br>
+                
+                <label>Small bio: </label><br>
+                <textarea cols="50" rows="4"></textarea><br><br>
+                
+                <table>
+                    <tr>
+                    <th><input class="but" name="submit" type="submit" value="Submit" /></th>
```
Comment (nawetimebomb) [inline 00-HTML-CSS-basics/vi.html]: Do not use short names for anything. Also you should've been using `<button>` instead of `<input>`. HTML5 things.

**Review by nawetimebomb — APPROVED** [review]
This is very very good. Much better. I approved because there are two minor issues and you can merge or fix it (I would always recommend fixing though). Anyway keep up the good work and continue enthusiastic about this!

File: 00-HTML-CSS-basics/vi.html (hunk 18 above)
Comment (agustinavighetti) [reply 00-HTML-CSS-basics/vi.html]: Fixed.

File: 00-HTML-CSS-basics/layout.css (hunk 17 above)
Comment (agustinavighetti) [reply 00-HTML-CSS-basics/layout.css]: I changed the names, are they ok now?

**Review by nawetimebomb — APPROVED** [review]
Merge it! Hahaha
